packages/wonder-blocks-tokens/src/tokens/sizing.ts

```diff
@@ -32,11 +32,15 @@ export const sizing = {
     size_120: pxToRem(12),
     size_140: pxToRem(14),
     size_160: pxToRem(16),
+    size_170: pxToRem(17),
```

Comment:
Note: I added these extra tokens after testing in webapp. There are some additional font-sizes on the LOHP and other places that I wanted to include in WB for minimal impact on teams. There are likely to be more, as there is a very large surface area for font-sizing in components. We could try to convince teams to use the standard tokens, though it will be a lot of work to do so!

Reply:
Actually, I see it the other way around. We should avoid adding these tokens to provide a more consistent experience, so I would strongly recommend not adding these tokens here and instead just changing the call sites to use WB tokens and/or leaving those as one-offs. The signal about those tokens being used is that those values were introduced when the design language was different around that time, and it would be best not to introduce these legacy tokens to WB.

Reply:
For the one-offs, would we want to export the `pxToRem` utility so that custom values can scale in rems too?

Reply:
Alright, I'll remove them from this PR and do something else for call-sites in webapp.

For the `pxToRem` utility, how would that be used if we did export it? Isn't that an internal tool for WB?

Reply:
Ah, in updating the font / sizing tokens I discovered 3 of these added values are in WB: `17px`, `22px`, and `26px`

- `HeadingMedium` at a small viewport size: font-size `22px` and line-height `26px`

```
  HeadingMedium: {
        [mobile]: {
            fontSize: sizing.size_220,
            lineHeight: sizing.size_260,
        },
    },
```

- `BodyMonospace`: font-size `17px` and line-height `22px` (this one is used a LOT in webapp)

```
BodyMonospace: {
        fontSize: sizing.size_170,
        lineHeight: sizing.size_220,
    },
```

- `Body`: line-height `22px` (also used a lot)

- `BodySerifBlock`: font-size `22px` but it's only used on an `sat-landing-package` / `summary-module.tsx` in webapp. Is that product used anymore?

Reply:
> For the pxToRem utility, how would that be used if we did export it? Isn't that an internal tool for WB?

Right now it is internal to WB since we don't export it!

If we were to export `pxToRem`, then consumers can import it and use it to convert px values to rem. Consumers should use the sizing tokens where they can, but if there are one off values not in the WB scale, they can convert it to rems using the utility which will use the same WB baseline (10). The utility would make it so if we have to change the rem baseline, the one-offs using the utility wouldn't need to change since it takes into account the WB defined baseline.

That said, exposing `pxToRem` could encourage more one-off values too. We don't have to export the utility, but thought I'd share the thought!

Reply:
Added some tokens with necessary font-size changes here to avoid one-offs. I think we could keep chatting about `pxToRem` and tackle it in a future PR, potentially?